api/HTMLMediaElement.json

```
@@ -3768,7 +3768,13 @@
               "version_added": "3.1"
             },
             "safari_ios": {
-              "version_added": "3"
+              "version_added": "3",
+              "partial_implementation": true,
+              "notes": [
+                "`volume` returns a value and is writeable; however, the value is always 1, and setting a value has no effect on the volume of the media object.",
```

Notes should be HTML formatted.  (Also slight spelling fix.)

```suggestion
                "<code>volume</code> returns a value and is writable; however, the value is always 1, and setting a value has no effect on the volume of the media object.",
```